README.skip.md

```diff
@@ -0,0 +1 @@
+this is developed based on AFL++ 60dc37a8cf09f8e9048e4b6a2204d6c90b27655a
```

Comment:
I think this README can be removed?
it would be good to have a description about what skipdet does though, who developed it etc. maybe in a file called docs/new_technology.md ? (in which future new tech can be added to.)
does not need to be in this PR, can also be added later.

Reply:
I forgot it, I'll add more details to the doc after paper is done.